wcivf/apps/elections/templates/elections/includes/_cancelled_election.html

```diff
@@ -1,27 +1,81 @@
 {% load markdown_deux_tags %}
+{% load humanize %}
+{% load static %}
+
 {% if object.metadata.cancelled_election.title %}
-    <h2>{{ object.metadata.cancelled_election.title }}</h2>
+    <h4>{{ object.metadata.cancelled_election.title }}</h4>
 {% else %}
-    <h2>Cancelled Election</h2>
+    {% if not object.contested %}
+        {% if object.winner_count == object.people.count %}
+            <h4>Uncontested Election</h4>
+
+            <p>This election was uncontested because the number of candidates who stood was equal to the number of available seats.
+                There {{ object.winner_count|pluralize:"is,are" }} {{ object.winner_count|apnumber }}
+                seat{{ object.winner_count|pluralize }} in {{ object.post.full_label }}, and only
+                {{ object.people.count|apnumber }} candidate{{ object.people|pluralize }}.</p>
+
+            <p>No votes will be cast, and the candidate{{ object.people|pluralize }} below ha{{ object.winner_count|pluralize:"s,ve" }} been automatically declared
+                the winner{{ object.people|pluralize }}.</p>
+
+            {% if object.display_as_party_list %}
+                {% include "elections/includes/_people_list_with_lists.html" with people=object.people %}
+            {% else %}
+                {% include "elections/includes/_people_list.html" with people=object.people %}
+            {% endif %}
+
+        {% elif object.winner_count > object.people.count %}
```

Comment:
This logic won't work if we want the "Uncontested and Rescheduled Election" text to render in the case of a no candidates & uncontested.

Reply:
Could you move the `{% if object.people %}` check further down to where it is used? E.g.:
```
{% elif object.winner_count > object.people.count %}
    <h4>Uncontested and Rescheduled Election</h4>

     <p>This election was uncontested because the number of candidates who stood was fewer than the number of available seats.<p>
     {% if object.people %}
         ....
```